### DISCUSSION TITLE
我有一篇避雷美国某项目的文章，怎么上传啊

### DISCUSSION THREAD

**Discussion by edchung777** [The-Run-Philosophy-Organization/run#365] [General]
我是美国移民法律相关人士，自己写了一篇想让大家避雷某项目的文章。。。用不来github ，哭哭。。。。

**Comment by ghost** [comment]
请给出文章链接

**Comment by the-run-philosophy** [comment]
你可以直接把文章贴在这里我们帮你提交；正确的做法是创建一个GITHUB账号（你已经有了），然后选择FORK，然后在自己FORK好的REPO里直接编辑；（包括新建文件，上传等），然后选择在你的REPO的主页找到“提交PR”按钮即可。
